doc/index.js.html

```diff
@@ -118,7 +121,16 @@ <h1 class="page-title">Source: index.js</h1>
 	 * @param {object} subscriberConfig.config - config object
 	 * @returns {Promise}
 	 */
-	subscribe: subscribeFunc
+	subscribe: subscribeFunc,
+
+	/**
+	 * Closes the connection to RabbitMQ.
```

Comment:
I think @sfryffdc told me once that despite the repo name, this repo is not rabbitmq specific, and works with any amqp. If so, then comments should probably not be RabbitMQ specific.